org.kde.digikam.yml

```diff
@@ -0,0 +1,243 @@
+id: org.kde.digikam
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: digikam
+desktop-file-name-suffix: ' (Beta)'
+finish-args:
+  - --device=all
+  - --filesystem=xdg-picrures
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=wayland
+  - --socket=x11
+  - --system-talk-name=org.freedesktop.UDisks2
+  # Set proper path for digiKam plugins
+  - --env=DK_PLUGIN_PATH=/app/lib/plugins/digikam
+rename-icon: digikam
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/libexec
+  - /lib/pkgconfig
+  - /share/doc
+  - /share/eigen3
+  - /share/man
+  - /share/pkgconfig
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+modules:
+  - name: digikam
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DDIGIKAMSC_CHECKOUT_PO=ON
+      - -DDIGIKAMSC_CHECKOUT_DOC=OFF
+      - -DDIGIKAMSC_COMPILE_PO=ON
+      - -DDIGIKAMSC_COMPILE_DOC=OFF
+      - -DENABLE_MYSQLSUPPORT=ON
+      - -DENABLE_INTERNALMYSQL=ON
+      # Currently disabled due compile error, will be fixed in the future
+      - -DENABLE_MEDIAPLAYER=OFF
+      - -DENABLE_DBUS=ON
+      - -DENABLE_APPSTYLES=ON
+      - -DENABLE_QWEBENGINE=ON
+      - -DENABLE_FACESENGINE_DNN=ON
+    sources:
+      - type: archive
+        url: https://download.kde.org/unstable/digikam/digikam-7.0.0-beta3.tar.xz
+        sha256: 97b2f768674afaaca6feb6e661d5c8fc6b88b08f4d4bbab8a6bcc94c19914893
+    modules:
+      - boost.yml
```

Comment:
Any reason for using a separate file for boost?

Reply:
No, any reasons. Move all contents of this file to the main manifest?

Reply:
Unless it's generated automatically, it's the preferred way.